SpatialGDK/Extras/schema/spawner.schema

```diff
@@ -3,6 +3,8 @@ package unreal;
 
 type SpawnPlayerRequest {
     string url = 1;
+    bytes unique_id = 2;
```

Comment:
Why bytes instead of string?

Reply:
This is serialized data. `string` implies a valid Unicode string.

Reply:
> String and bytes fields are both treated as bytes in the C API, because they are both treated in the same way over the wire. In SpatialOS, the only difference is how code is generated for them in the C++, C#, and Java SDK, and how they’re visualized in the inspector.

https://docs.improbable.io/reference/13.5/capi/serialization#strings-and-bytes

I think it should be string due to it usually being human readable.